src/RefreshManager.m

```diff
@@ -33,6 +33,7 @@
 #import "NSNotificationAdditions.h"
 #import "VTPG_Common.h"
 #import "FeedItem.h"
+#import <QuartzCore/QuartzCore.h>
```

Comment:
I don't think we should be adding new UI to RefreshManager (and GoogleReader). This functionality would be better moved into another class.